### DISCUSSION TITLE
TSC: Creating a better definition on evaluation criteria for activation

### DISCUSSION THREAD

**Discussion by brunodom** [finos/technical-oversight-committee#68] [General]
Hi TSC members,

As we previously discussed (https://github.com/finos/technical-steering-committee/pull/67), I would like to open up this "thread" to bring more clarity on what it means for activation purposes:

- Usefulness
- Community
- Roadmap

IMHO, let's start it with the end goal in mind. If we want the activation project to succeed, we need to properly assess the gap between the project's candidate and the desired outcome. So, here are some of my comments as a starting point to set some principles - if we all agree on the principles, we may move to more specifics of each:

- **Usefulness:** How likely will other financial institutions adopt this project? e.g., if there is something specific to the FSI or more attractive open-source/commercial and off-the-shelf solutions are available, it may not be worth investing in the project. E.g., TCP/IP stack is extremely important for any connected application (usefulness). However, it is widely available on virtually all OS and not specific to FSI.
- **Community:** Having been operating with the open community for some time can also give us some sense of how much adoption/acceptance the project may have. I see this as complementary to the "usefulness." Suppose they need to be operating more openly. At least it is good to understand how opening it can affect the project's future. E.g., you may have a project that wasn't open because it is highly coupled with specific platforms/architecture, and opening would be no adoption or starting from square zero.
- **Roadmap:** It may give us a sense of maturity and continuity. Without a roadmap, the project may have a short life and sure restrict adoption in most cases.

I hope that I provoked some health discussions so that we, the TSC, provide a path for projects to succeed instead of a simple gatekeeper.

My two cents!
-Bruno Domingues

**Comment by maoo** [comment]
Thanks @brunodom - I personally like your proposal, and I'm curious to hear feedback from the rest of the TSC.

As soon as there's more consensus built, I'd suggest to raise a Pull Request on https://community.finos.org/docs/governance/software-projects/stages/active/ and reflect changes there, agreed?

**Comment by eminty69** [comment]
These principles make sense to me but I'd probably add one more.  When the project was onboarded there would have been some goals defined in terms of direction, possible integration with other existing projects, etc.  I'd like to see this covered by the activation. Not sure what to call it - growth?

The other thing I'd like to see is the project maintainers giving us their take on how the project aligns with the principles.  These aren't yes/no things and understanding the maintainers perspective will give a lot of context.

**Reply by maoo** [reply]
I like the idea of introducing the concept of growth; maybe it could be one with the Roadmap, i.e. "Roadmap & Growth" ?

Also 👍 on the project maintainer's take; maybe we could draft a template for project maintainers to apply for activation ?

**Comment by yassinelaouni** [comment]


**Reply by yss-fgggg** [reply]
you are good
